modules/builtin/src/actions/getConversationVariable.js

```diff
@@ -0,0 +1,16 @@
+/**
+ * Get a variable from the conversation
+ * @title Get conversation variable
+ * @category Storage
+ * @author Botpress, Inc.
+ * @param {string} name - The name of the variable
+ * @param {string} output - The state variable to ouput to
+ */
+const getConversationVariable = async (name, output) => {
```

Comment:
is this related to your module ? If no please submit a different PR, we try to keep pull requests as scoped as possible

Reply:
@EFF this is necessary to get the information saved in the kvs at QNA setup.ts file, for some reason i didn't find a function to get info from conversation variables, please let me know if i missed something